attest/application_key.go

```diff
@@ -101,3 +102,8 @@ func (k *Key) Marshal() ([]byte, error) {
 func (k *Key) CertificationParameters() CertificationParameters {
 	return k.key.certificationParameters()
 }
+
+// PubPrivBlobs returns private and public blobs to be used by tpm2.Load().
+func (k *Key) PubPrivBlobs() ([]byte, []byte, error) {
```

Comment:
Want to just call this "Blobs()"? Or "Export()"?

Reply:
Done